docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -49,6 +49,26 @@ To change the width of a column, use the `width` property available in `ColDef`.
 
 {{"demo": "pages/components/data-grid/columns/ColumnWidthGrid.js", "bg": "inline"}}
 
+## Column fluid width
```

Comment:
I think that it should be part of the generic column width concern. 

```suggestion
### Fluid width
```

Reply:
I wonder about 
```suggestion
### Flex width
```

Reply:
I was wondering how to structure it but, yes having it as a subsection of the `Column width` does look better.

Reply:
I'm raising it because I want on using a flatter structure than ag-grid for the documentation. We will probably in the future need to break it further down as we add more features to the component.